.github/workflows/promptflow-sdk-cli-test.yml

```diff
@@ -79,7 +79,7 @@ jobs:
         az account show
         python "../../scripts/building/run_coverage_tests.py" `
           -p promptflow `
-          -t ${{ github.workspace }}/src/promptflow/tests/sdk_cli_azure_test ${{ github.workspace }}/src/promptflow/tests/sdk_cli_test `
+          -t ${{ github.workspace }}/src/promptflow/tests/sdk_cli_test `
```

Comment:
why is this included at the beginging?

Reply:
previously, we only have one CI workflow for SDK/CLI, so it contains both `sdk_cli_test` and `sdk_cli_azure_test`; currently, I think we'd better split them into two.